BeerCatalogFullstack/FrontEnd/src/components/favouritesPage/FavouritesList/FavouritesList.jsx

```diff
@@ -28,12 +33,20 @@ export default class FavouritesList extends React.Component {
         }
     }
 
-    onDelete = (item) => {
-        localStorageHelper.deleteItem(item);
+    onDelete = async (item) => {
         const { Beers } = this.state;
+
+        await favouritesServices.deleteItem(this.context.userId, item);
+        const result = await favouritesServices.getItems(this.context.userId);
+        const beers = result.map(async (beerId) => {
+            return beerServices.getByID(beerId);
+        });
+        Promise.all(beers).then((beersArray) => this.setState({ Beers: beersArray }));
```

Comment:
Could you do this without calling the service?

Reply:
Hmm, it seems that you do it below. In that case, do you need to call service at all?